modules/loader-utils/src/lib/file-provider/browser-file.ts

```diff
@@ -21,39 +21,39 @@ export class BrowserFile implements FileProvider {
    * @param lenght Length of read data
    */
   private async getBytesFromFile(start: number, lenght: number): Promise<ArrayBuffer> {
-    let reader = new FileReader();
+    const reader = new FileReader();
```

Comment:
- We already have blob-file.ts that handles this. 
- AFAIK, there is no advantage to use FileReader over `await Blob.arrayBuffer();`